### PR TITLE
Apply minor polish to core classes

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: core/src/main/java/io/smallrye/context/SmallRyeContextManager.java [hunk 1]
```diff
@@ -246,15 +245,13 @@ public SmallRyeThreadContext allClearedThreadContext() {
...
+        private final List<ThreadContextProvider> contextProviders = new ArrayList<>();
+        private final List<ContextManagerExtension> contextManagerExtensions = new ArrayList<>();
         private ExecutorService defaultExecutorService;
 
         @Override
         public Builder withThreadContextProviders(ThreadContextProvider... providers) {
-            for (ThreadContextProvider contextProvider : providers) {
-                contextProviders.add(contextProvider);
-            }
+            contextProviders.addAll(Arrays.asList(providers));
```
Comment (FroMage) [inline core/src/main/java/io/smallrye/context/SmallRyeContextManager.java:254]: Are you sure this is better? It creates an intermediary list…

**Review by FroMage — COMMENTED** [review]
Thanks, added a question

File: core/src/main/java/io/smallrye/context/SmallRyeContextManager.java (hunk 1 above)
Comment (geoand) [reply core/src/main/java/io/smallrye/context/SmallRyeContextManager.java:254]: Oh right. It's definitely not better. I'll remove it

Comment (geoand) [reply core/src/main/java/io/smallrye/context/SmallRyeContextManager.java]: Fixed

**Review by FroMage — DISMISSED** [review]

**Comment by geoand** [comment]
I pushed another tiny change

**Review by FroMage — APPROVED** [review]

**Pull request merged**